### PR TITLE
Update bower.json

### PR DESCRIPTION
**Pull request by chaitu87**
wrong filename.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "moment-timezone",
   "description" : "Parse and display moments in any timezone.",
   "license" : "MIT",
-  "main": "builds/moment-timezone-with-data-2010-2020.js",
+  "main": "builds/moment-timezone-with-data-2010-2022.js",
```
Comment (tommygnr) [inline bower.json]: @chaitu87 this is still wrong. It should be 2012-2022.

**Comment by tommygnr** [comment]
Refs #466

**Comment by marwahaha** [comment]
@chaitu87 could you sign the CLA?

**Comment by marwahaha** [comment]
This has been fixed.

**Pull request closed without merging**